code/game/machinery/doors/airlock.dm

```diff
@@ -570,38 +570,61 @@ About the new airlock wires panel:
 /obj/machinery/door/airlock/attack_ghost(mob/user)
 	if(panel_open)
 		wires.Interact(user)
-	ui_interact(user)
+	tgui_interact(user)
 
 /obj/machinery/door/airlock/attack_ai(mob/user)
-	ui_interact(user)
+	tgui_interact(user)
 
-/obj/machinery/door/airlock/ui_interact(mob/user, ui_key = "main", datum/nanoui/ui = null, force_open = 1)
-	ui = SSnanoui.try_update_ui(user, src, ui_key, ui, force_open)
+/obj/machinery/door/airlock/tgui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = FALSE, datum/tgui/master_ui = null, datum/tgui_state/state = GLOB.tgui_default_state)
```

Comment:
You change `force_open = 1` to `force_open = FALSE`. Was this meant to be `TRUE`?